src/migrations/1608030374841_namespaces.ts

```diff
@@ -91,7 +91,14 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
     },
   });
 
-  pgm.createIndex('namespaces', 'index_block_hash', { method: 'hash' });
-  pgm.createIndex('namespaces', 'microblock_hash', { method: 'hash' });
-  pgm.createIndex('namespaces', [{ name: 'ready_block', sort: 'DESC' }]);
+  pgm.createIndex('namespaces', [
```

Comment:
This PR is intended to be a v4.x release right? If so, need to move these changes into new migration files.

But also, if this is for v4.x, will the changes fix the already-ingested BNS names, or only fix new BNS names? If only new ones, then is there a good reason to try to get this into v4 instead of develop/v5?

Reply:
The idea for this PR is for it to become v5 by itself... a completely new replay is required. Develop would become v6. Wdyt?

Reply:
Sounds good to me!